ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Selection-Tool/selection-tool.spec.ts

```diff
@@ -0,0 +1,424 @@
+    await page.keyboard.press('Control+a');
+    await page.keyboard.down('Shift');
+    for (let i = 0; i < 10; i++) {
+      await page.keyboard.press('ArrowRight');
+    }
+    await page.keyboard.up('Shift');
+  });
+
+  test('(100px to Left with Shift key) Structure Movement with Arrow Keys (10px move)', async ({
+    page,
+  }) => {
+    /*
+    Test case: EPMLSOPKET-15512
+    Description: The selected structure should move 10 pixel in the corresponding 
+    direction with each key press with Shift key. In this test to 100px Left.
+    */
+    await openFileAndAddToCanvas('two-benzene-with-atoms.ket', page);
+    await takeEditorScreenshot(page);
+    await page.keyboard.press('Control+a');
+    await page.keyboard.down('Shift');
+    for (let i = 0; i < 10; i++) {
+      await page.keyboard.press('ArrowLeft');
+    }
+    await page.keyboard.up('Shift');
+  });
+
+  test('Field value text when placed on a structure becomes hard to access', async ({
+    page,
+  }) => {
+    /*
+    Test case: EPMLSOPKET-12974
+    Description: User can easily select 'Field value' text and move to desired location.
+    */
+    const pointx = 580;
+    const pointy = 400;
+
+    const pointx1 = 750;
+    const pointy1 = 300;
+    await openFileAndAddToCanvas('two-benzene-with-atoms.ket', page);
+    await selectLeftPanelButton(LeftPanelButton.S_Group, page);
+    await clickOnAtom(page, 'C', 0);
+    await fillFieldByPlaceholder(page, 'Enter name', 'Test');
+    await fillFieldByPlaceholder(page, 'Enter value', '33');
+    await pressButton(page, 'Apply');
+
+    await selectLeftPanelButton(LeftPanelButton.RectangleSelection, page);
+    await page.getByText('33', { exact: true }).click();
+    await dragMouseTo(pointx, pointy, page);
+    await takeEditorScreenshot(page);
+
+    await page.getByText('33', { exact: true }).click();
+    await dragMouseTo(pointx1, pointy1, page);
+  });
+});
+
+test.describe('Selection tools', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+  });
+
+  test('Selection tools is not change when user press ESC button', async ({
+    page,
+  }) => {
+    /*
+    Test case: EPMLSOPKET-10074
+    Description: If user presses esc, then last chosen selected tool must be 
+    selected and pressing esc doesn't choose another mode of selection tool
+    */
+    await selectRing(RingButton.Benzene, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await page.keyboard.press('Control+a');
+    for (let i = 0; i < 2; i++) {
+      await page.keyboard.press('Escape');
+    }
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('Verify removal of current flip and rotation buttons in the left toolbar', async ({
+    page,
+  }) => {
+    /*
+    Test case: EPMLSOPKET-15511
+    Description: The flip and rotation buttons are no longer present in the left toolbar.
+    */
+    await expect(page).toHaveScreenshot({
+      clip: { x: 0, y: 0, width: 800, height: 600 },
```

Comment:
Why do you screenshot this area?
May be you can add testid for some specific DOM-node, which you want to screenshot?